### PR TITLE
OXO TITAN - Visualize predefined agent groups when selected in  the new scan page.

### PR DESCRIPTION
**Pull request by adnaneserrar**
[Screencast from 01 غشت, 2024 +01 17:43:39.webm](https://github.com/user-attachments/assets/350c9aab-2bef-49f6-860a-9dc374070f53)

### CODE REVIEW

File: project/scans/components/AgentGroupSelect.vue [hunk 1]
```diff
@@ -4,37 +4,77 @@
...
-          v-bind="props"
-          :index="index"
-          title=""
-        >
-          <v-list-item-title>{{ item.raw.key }}</v-list-item-title>
-          <v-list-item-subtitle>{{ item.raw.description }}</v-list-item-subtitle>
-        </v-list-item>
-      </template>
-    </v-select>
+    <v-form>
```
Comment (3asm) [inline project/scans/components/AgentGroupSelect.vue:7]: In the video, some previous buttons had an icon, others didn't.

File: project/scans/components/AgentGroupSelect.vue [hunk 2]
```diff
@@ -111,20 +170,93 @@ export default defineComponent({
...
+     * Fetch agent groups.
      */
     async getAgentGroups(scanner: Scanner): Promise<void> {
       try {
         this.agentGroups = await this.agentGroupService.getAgentGroups(scanner, this.agentGroupAssetType)
       } catch (e: any) {
         this.reportError(e?.message || 'Error fetching agent groups')
       }
+    },
+    async createScan(): Promise<void> {
```
Comment (3asm) [inline project/scans/components/AgentGroupSelect.vue:197]: Doctstring.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: project/scans/components/AgentGroupSelect.vue [hunk 3]
```diff
@@ -4,37 +4,77 @@
...
+        class="min-height-300"
+      >
+        <div class="d-flex align-start">
+          <div class="width-30">
+            <div class="d-flex align-center justify-space-between tab-header">
+              <div>
+                <v-icon start>
+                  mdi-hexagon-multiple-outline
+                </v-icon>
+                Existing Agent Groups
```
Comment (3asm) [inline project/scans/components/AgentGroupSelect.vue:28]:
```suggestion
                Agent Groups
```

**Review by 3asm — CHANGES_REQUESTED** [review]
Very nice work getting out this fast.

File: project/scans/components/AgentGroupSelect.vue [hunk 4]
```diff
@@ -4,37 +4,77 @@
...
-        </v-list-item>
-      </template>
-    </v-select>
+    <v-form>
+      <div class="d-flex align-center mb-3">
+        <p class="mr-1 text-grey">
+          Run a scan with
+        </p>
+        <p class="ml-1 text-grey">
+          agent group
```
Comment (ybadaoui-ostorlab) [inline project/scans/components/AgentGroupSelect.vue:13]: Agent group

File: project/scans/components/AgentGroupSelect.vue (hunk 3 above)
Comment (adnaneserrar) [reply project/scans/components/AgentGroupSelect.vue]: Screenshot from 2024-08-01 18-32-06

**Review by 3asm — APPROVED** [review]

**Pull request merged**